src/switch/switch.stories.ts

```diff
@@ -10,7 +10,10 @@ storiesOf("Switch", module).addDecorator(
 )
 	.addDecorator(withKnobs)
 	.add("Basic", () => ({
-		template: `<ibm-switch [disabled]="disabled"></ibm-switch>`,
+		template: `
+		<ibm-switch [disabled]="disabled"></ibm-switch>
+		<ibm-switch [size]="'sm'"></ibm-switch>
```

Comment:
Could you add the `[disabled]="disabled"` attribute as well? that way both will respect the "disabled" knob

Reply:
Done.